dotnet/src/SemanticKernel.Core/Data/VolatileVectorRecordStore.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Service for storing and retrieving vector records, that uses an in memory dictionary as the underlying storage.
+/// </summary>
+/// <typeparam name="TRecord">The data model to use for adding, updating and retrieving data from storage.</typeparam>
+[Experimental("SKEXP0001")]
```

Comment:
Should not it be SKEXP0020 ? Same for VolatileVectorRecordStoreOptions class.
<img width="350" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/68852919/7c8d86ff-a997-435c-a86f-2671f74c0c57">

Reply:
As discussed, all the existing memory code in core has 0001 as the category.  Let's discuss more with the rest of the team and address as a separate PR if needed.